runtime/filetype.vim

```diff
@@ -710,16 +710,19 @@ au BufNewFile,BufRead auto.master		setf conf
 au BufNewFile,BufRead *.mas,*.master		setf master
 
 " Forth
-au BufNewFile,BufRead *.ft,*.fth		setf forth
+au BufNewFile,BufRead *.ft,*.fth,*.4th		setf forth
 
 " Reva Forth
 au BufNewFile,BufRead *.frt			setf reva
 
 " Fortran
 if has("fname_case")
-  au BufNewFile,BufRead *.F,*.FOR,*.FPP,*.FTN,*.F77,*.F90,*.F95,*.F03,*.F08	 setf fortran
+  au BufNewFile,BufRead *.F,*.FOR,*.FPP,*.FTN,*.F77,*.F90,*.F95,*.F03,*.F08	setf fortran
```

Comment:
Is it intentional that *.F is still mapped to Fortran only?

Reply:
It is on the grounds that:

- Fortran has some conventions for uppercase extensions that makes them meaningful.
- We don't support uppercase for any other Forth extensions or other filetypes in general.
- SwiftForth (a notable user of `*.f`) doesn't implicitly recognise `*.F` as it does `*f`.

Is there some history of intentionally capitalised extensions?

Reply:
Not that I am aware of. I was simply unaware of Fortran conventions.